components/molecules/TableHeader/table-header.tsx

```diff
@@ -37,7 +69,8 @@ const TableHeader = ({
           ""
         )}
         {onSearch ? (
-          <Search placeholder={`Search ${title}`} className="max-w-full text-sm py-1.5" name={"query"} onSearch={onSearch} />
+          <Search placeholder={`Search ${title}`} className="max-w-full text-sm py-1.5" name={"query"} onSearch={onSearch}
```

Comment:
This need to be debounced so it doesn't hammer the search API. Ideally if the value hasn't changed after 250ms, then send the request to the API

Reply:
We can't use the search API at all, since it's just 30/60 requests per hour; using those requests in here would disable a user being able to search on GitHub. Even in a very good debounce implementation, a user could waste 2 or 3 of those requests in one single query.

Reply:
Isn't it per minute like described here? 
![Screenshot_20230226_154304](https://user-images.githubusercontent.com/52013393/221414128-7ef44b24-47e3-46ff-a039-f50ed2745370.jpg)

Reply:
good catch, I never read "per minute" and thought everything is per hour 😓